IBPSA/Fluid/Actuators/Valves/TwoWayPolynomial.mo

```diff
@@ -0,0 +1,43 @@
+within IBPSA.Fluid.Actuators.Valves;
+model TwoWayPolynomial "Two way valve with polynomial characteristic"
+  extends IBPSA.Fluid.Actuators.BaseClasses.PartialTwoWayValveKv(
+    phi=l + pol_y*(1 - l));
+
+  parameter Real[:] c "Polynomial coefficients, starting with fixed offset";
```

Comment:
Do you have an example setting for `c` for a few valves? This should better be in a data record so that a library of valve coefficients, together with their leakage and kV_s values, can be provided to the user.

Reply:
I currently have only one example.

Using records makes sense, but not if we only do it for the polynomial model? Then all valves should be refactored. Unless you want to use the polynomial model as a 'one size(/model) fits all' model which is used to build up a database of valve implementations, including linear valves etc.
In this case I'd propose to make a new model `TwoWay` or `TwoWayRecord` that extends `TwoWayPolynomial` and that declares the record and propagates relevant parameters.

Reply:
@mwetter : please see my reply :)